pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.github.bric3.maven</groupId>
     <artifactId>cql-maven-plugin</artifactId>
-    <version>0.4</version>
+    <version>0.5</version>
```

Comment:
If you want you can bump to 1.0.0, as it's a breaking change. What do you think?

Reply:
Yes it makes sense 👍

Reply:
Also it might be a good opportunity to switch to a full semver.